src/app/modals/encryptwallet/encryptwallet.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="container">
+  <div class="row">
+    <div *ngIf="!password" class="col-xs-12 text-center">
+      <h4>Please enter <b>a password</b> to encrypt your wallet.</h4>
+      <h4><b>Do NOT forget this password or you will be unable to access your coins.</b></h4>
+    </div>
+
+    <div *ngIf="password" class="col-xs-12 text-center">
+      <h4>Please enter <b>your password again</b> to confirm that you've typed the right password.</h4>
+      <h4><b>Do NOT forget this password or you will be unable to access your coins.</b></h4>
+    </div>
+
+  </div>
+</div>
+
+<hr class="margin-top-100px">
+
+<app-password
+  #passwordElement
+  [buttonText]="'ENCRYPT WALLET'"
```

Comment:
Change to
`buttonText="ENCRYPT WALLET"`

Reply:
You'd think that would work right, but I tried it a few months ago and it didn't for some odd reason.
Will check.

Reply:
Checked up on this, ' ' is required.

Reply:
Have a look at createwallet.component.html :)